src/renderer/components/dex/OrderForm.vue

```diff
@@ -269,21 +269,30 @@ export default {
       }
       return !this.$store.state.orderQuantity || !this.$store.state.orderPrice || this.$isPending('placeOrder');
     },
+    orderTypes() {
+      const list = [{
```

Comment:
This can be instantiated once before the export statement.

Reply:
Don't understand what you mean here.

Reply:
Rather than creating this const `list` on each call to this method, just create it once on L83 (with a new line before it for proper formatting).


```
<script>
import { BigNumber } from 'bignumber.js';
import AphOrderConfirmationModal from '../modals/OrderConfirmationModal';
import AphDepositWithdrawModal from '../modals/DepositWithdrawModal';

const orderTypesList = [ // <--- here
  {
    label: 'Limit',
    value: 'Limit',
  },
];
let loadHoldingsIntervalId;

export default {

Reply:
But what about when they switch back and forth between ledger and a non ledger wallet? I'm trying to have the market option available for non ledger wallets and hidden when they are using a ledger.

Reply:
```
if (this.canPlaceMarketOrder) {
  return _.concat(ORDER_TYPES_LIST, [
    {
      label: 'Market',
      value: 'Market',
    }, 
  ])
}

return ORDER_TYPES_LIST;
```